modules/understanding-upgrade-channels.adoc

```diff
@@ -7,104 +7,98 @@
+
+Beginning with {product-title} 4.10 whenever update recommendations are removed from supported channels they are replaced with Conditional Updates which declare one or more known risks. Each known risk may apply to all clusters or only clusters matching certain conditions such as having `Platform` set to `None` or the CNI provider set to `OpenShiftSDN`. The Cluster Version Operator (CVO) continually evaluates known risks against the current cluster state. When no risks match, the update is recommended. If the risk matches those updates are listed as Supported But Not Recommended Updates and a reference link is provided which helps the cluster admin decide if they would like to accept the risk and update anyway.
+
+[role="_additional-resources"]
+.Additional resources
+* For more information, see xref:../updating/updating-cluster-cli.adoc#update-conditional-upgrade-pathupdating-cluster-cli[Updating along a conditional upgrade path]
 
-ifdef::openshift-origin[]
-The presence of an update recommendation in the `stable-4` channel at any point is a declaration that the update is supported. While releases will never be removed from the channel, update recommendations that exhibit serious issues will be removed or made conditional from all channels. When an update recommendation is supported, it remains supported for the life of {product-version}, even if the update recommendation is later dropped or made conditional.
-endif::openshift-origin[]
 
 ifndef::openshift-origin[]
 
 [id="fast-stable-channel-strategies_{context}"]
-== Fast and stable channel use and strategies
+== Choosing the correct channel for your cluster
 
-The `fast-{product-version}` and `stable-{product-version}` channels present a choice between receiving general availability releases as soon as they are available or allowing Red Hat to control the rollout of those updates. If issues are detected during rollout or at a later time, upgrades to that version might be blocked in both the `fast-{product-version}` and `stable-{product-version}` channels, and a new version might be introduced that becomes the new preferred upgrade target.
+Choosing the appropriate channel involves two decisions.
+
+First, select the minor version you wish for your cluster to upgrade to. Selecting a channel which matches your current version ensures that you only apply z-stream updates and do not receive feature updates. Selecting an available channel which has a version greater than your current version will ensure that after one or more updates your cluster will have updated to that version. Your cluster will only be offered channels which match its current version, the next version, or the next EUS version. Due to the complexity involved in planning upgrades between versions many minors apart we do not offer channels which assist in planning upgrades beyond a single EUS-to-EUS update.
```

Comment:
`wish` -> `want`

Change
`Due to the complexity involved in planning upgrades between versions many minors apart we do not offer channels which assist in planning upgrades beyond a single EUS-to-EUS update` to 

`Due to the complexity involved in planning upgrades between versions many minors apart, channels that assist in planning upgrades beyond a single EUS-to-EUS update are not offered.`

Reply:
Ok, what do you think about having the `Due to the complexity...` bit as an admonition?

Reply:
I went ahead with a NOTE admonition.